pkg/fleet/installer/packages/ssi/status.go

```diff
@@ -0,0 +1,14 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package ssi adds methods to check the status of the APM Single Step Instrumentation
+package ssi
+
+// APMInstrumentationStatus contains the instrumentation status of APM Single Step Instrumentation.
+type APMInstrumentationStatus struct {
```

Comment:
For windows will it be possible to extend this with other fields we might need ? Like IISInstrumented for instance.

Reply:
I added a new field that reports the instrumentation method(s). Today it's "host" and/or "docker", we'll be able to add IIS in the future